lib/customer-deposit-wallet/customer-deposit-wallet.cabal

```
@@ -1,7 +1,7 @@
 cabal-version:      3.6
 build-type:         Simple
 name:               customer-deposit-wallet
-version:            0.1.0.0
+version:            2024.5.5
```

```suggestion
version:            0.2024.5.5
```

🤔 My original conception for the `customer-deposit-wallet` was to be more of a library than an executable. Since we decided to link it into the `cardano-wallet` executable, I think it's appropriate to use the "library version numbering" here. But I don't mind either way.

---

`0.` added